docs/pull_request_template.md

```diff
@@ -1,7 +1,7 @@
 ## Purpose
 _Briefly describe the purpose of the change, and/or link to the JIRA ticket for context_
 
-Fixes LPA-####
+Fixes LPAL-####
```

Comment:
do we want this in a different PR?

Reply:
was literaly one character change spotted, so fixed whilst here.